pom.xml

```
@@ -42,6 +42,7 @@
     <modules>
         <module>api</module>
         <module>omod</module>
+        <module>omod-fhir</module>
```

While Maven will sort this out, this should actually be:

```suggestion
        <module>omod-fhir</module>
        <module>omod</module>
```

Since the OMOD depends on the FHIR stuff